### PR TITLE
Add short options for codebase and codebase-create

### PR DESCRIPTION
**Pull request by ChrisPenner**
## Overview

Adds `-c` for `--codebase` and `-C` for `--codebase-create`, I got tired of typing these every time 😓

### CODE REVIEW

File: unison-cli/unison/ArgParse.hs [hunk 1]
```diff
@@ -244,15 +245,17 @@ codebasePathParser :: Parser (Maybe CodebasePathOption)
 codebasePathParser = do
     optString <- optional . strOption $
          long "codebase"
-      <> metavar "codebase/path"
+      <> short 'c'
+      <> metavar "CODEBASE/PATH"
```
Comment (aryairani) [inline unison-cli/unison/ArgParse.hs:249]: what's with the all caps metavar vs lower case?

Comment (ChrisPenner) [reply unison-cli/unison/ArgParse.hs:249]: The rest of the metavars in this file are uppercase;
It's a relatively common convention in modern cli's, open for debate of course, but I find it helps make it clear when something is supposed to be replaced.

I've also commonly seen `<METAVAR>` with angle brackets.

I'm open to whatever we think is best, but this is how stack does it 😄 

After a quick sampling,

* git uses `<metavar>`
* rg uses `<METAVAR>`
* stack uses `METAVAR`
* fzf uses `METAVAR`

So looks like conventions are all over the place these days 🤷🏼‍♂️ ;
The most important is that we're consistent I think. If I had to pick I'd prefer `<METAVAR>`, but right now the other unison flags use `METAVAR`

Comment (aryairani) [reply unison-cli/unison/ArgParse.hs:249]: Interesting...

**Review by aryairani — APPROVED** [review]

**Comment by pchiusano** [comment]
I'm cool with these, but I'd stay away from using short flags in official docs, they're really more of a convenience thing when you don't feel like typing. :)

**Review by pchiusano — APPROVED** [review]

**Pull request merged**